### PR TITLE
Improve documentation on packaging emscripten

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by sbc100** [comment]
@svenstaro , @chenrui, I've listed the homebrew and archlinux packages that you maintain here, as those seem to the ones that are seeing the most uses.  Do you mind if I include your names and/or email here?

I'm hoping this document can be act as point of contact and coordination going forward.

**Comment by svenstaro** [comment]
Ok sure

**Comment by sbc100** [comment]
Oops that should have been @chenrui333

File: docs/packaging.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+Packaging Emscripten
+====================
+
+This document is designed to aid in packaging emscripten.  For example within
+linux distributions, or other downstream SDKs.
+
+Firstly, we provide and install script in `tools/install.py`.  This allows
```
Comment (kripken) [inline docs/packaging.md:7]: `and => an`

File: docs/packaging.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+the stable LLVM releases.  It requires the a build of LLVM from top-of-tree,
+or at least very close ot it.  This means that depending on a packaged version
+of LLVM is unlikely to work.
+
+The core
+[DEPS](https://chromium.googlesource.com/emscripten-releases/+/refs/heads/master/DEPS)
+file in the `emscripten-releases` repository contains all the information about
+about the varisou repositories that go into an emscripten release. This is the
+repo used by our CI to build and test emscripten.  To find out which revisions
+when into a given release you need to check out the DEPS file that the revision
```
Comment (kripken) [inline docs/packaging.md:28]: `when => went`

File: docs/packaging.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+-----
+
+If you package emscripten and don't want the ports system to be used, you can
+simply delete the `tools/ports/` directory.
+
+Prebuilt libraries
+------------------
+
+Ideally a packaged installation can include a fully populated cache directory
+contains pre-built libraries.   If you want to do this you can use `./embuilder
```
Comment (kripken) [inline docs/packaging.md:114]: `contains => containing`

File: docs/packaging.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+build ALL` to populate the cache directory.  You can them ship the `cache`
+directory inside the emscripten directory.  When shipping the cache directory on
+a multi-user system where users cannot modify the `cache` you need to be sure
+that all possible configurations of the libraries are build.  Currently that
```
Comment (kripken) [inline docs/packaging.md:118]: `build => built`

**Comment by chenrui333** [comment]
> Oops that should have been @chenrui333

Totally, happy to help. :)

**Pull request merged**